README.md

```diff
@@ -0,0 +1,24 @@
+# Custom paymaster tutorial
+
+Code for the "Build a custom paymaster" tutorial from the [zkSync v2 documentation](https://v2-docs.zksync.io/dev/).
+
+You can find a full step-by-step guide to build this project [in this article](https://v2-docs.zksync.io/dev/tutorials/custom-paymaster-tutorial.html#prerequisite).
+
+## Installation and compilation
+
+You need Node.js and Yarn.
+
+Install all dependencies with `yarn`.
```

Comment:
I'd rather say "Install all dependencies with `yarn add` "

Reply:
fixed in latest commit